pkgs/development/libraries/freetype/default.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-R2byAVfMTPDNKS+Av5F/ktHEObJDrDAY3r9rkUDEGn8=";
   };
 
-  propagatedBuildInputs = [ zlib bzip2 libpng ]; # needed when linking against freetype
+  propagatedBuildInputs = [ zlib bzip2 brotli libpng ]; # needed when linking against freetype
```

Comment:
Do we know that packages that depend on FreeType must also depend directly on brotli?

Reply:
Blender apparently?

Reply:
Not sure it's necessary, but i'm out of my element here. I got Blender 3.4 working in my repo (a couple weeks ago, on release) by just changing `nativeBuildInputs`.. not sure if it propagates? Eg, from my config:

```
          freetype_brotli = prev.freetype.overrideAttrs (old: rec {
            nativeBuildInputs = old.nativeBuildInputs ++ [ prev.brotli ];
          });
          
          blender_latest = (prev.blender.overrideAttrs (old: rec {
            pname = "blender";
            version = "3.4.0";
            src = prev.fetchurl {
              url = "https://download.blender.org/source/${pname}-${version}.tar.xz";
              sha256 = "sha256-KdBpOcUQ43NHEboW8nqtzMbtVRzhRq6eN6zUosohNJo=";
            };               
            nativeBuildInputs = old.nativeBuildInputs ++ [ prev.libepoxy freetype_brotli ];
          })).override {
            cudaSupport = true;
          };
```

Reply:
In that case, we would need to introduce a `freetype_brotli` and see all packages that need it. I am also unable to make a decision on this.